packages/model-viewer-effects/package.json

```diff
@@ -73,11 +73,11 @@
     "postprocessing": "^6.30.0"
   },
   "peerDependencies": {
-    "@beilinson/model-viewer": "^3.1.4",
+    "@google/model-viewer": "^3.1.0",
```

Comment:
I think our latest is 3.0.2

Reply:
yes all tests work and are passing locally, the CI is downloading the latest release but i've already bumped the model-viewer version in this PR, do you want me to change it back for now?

Reply:
Yes, I bump versions when I do a release - I think that may be confusing the CI.